tests/Morphir/Scala/BackendTests.elm

```diff
@@ -6,7 +6,7 @@ import Morphir.IR.Name as Name
 import Morphir.IR.Path as IRPath
 import Morphir.IR.Type as IRType
 import Morphir.Scala.AST as Scala
-import Morphir.Scala.Backend as Backend
+import Morphir.Scala.Feature.Core as Core
```

Comment:
Now that you changed the module name that is being tested you should change the testing module name as well to avoid confusion.

Reply:
This has been renamed now